src/LatinCharValidator.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace CrazyFactory\Validation;
+
+class LatinCharValidator
+{
+    public static function isValid(string $txt): bool
+    {
+        return empty($txt) || preg_match('/^[\p{Latin}\p{Common}]+$/u', $txt) > 0;
```

Comment:
- isValid === True when parameter is empty.

Reply:
yep. that's intended.